modules/cron/docs/cron.md

```diff
@@ -80,3 +88,35 @@ follow the appropriate links to execute the cron jobs you want. The
 page will take a while loading, and eventually show a blank page.
 
 
+### With CLI
+
+You can invoke cron functionality by running
+`/var/simplesamlphp/modules/cron/bin/cron.php` and providing a tag
+with the `-t ` argument.
+
+It is strongly recommended that you run the cron cli script as the
+same user as the web server.  Several cron hooks created files and
+those files may have the wrong permissions if you run the job as root.
+
+**note:** Logging behavior in SSP when running from CLI varies by
+version. The latest version logs to PHP's error log and ignores any
+logging configuration from `config.php`
+
+Below is an example of invoking the script. It will:
+
+* Run a command as the apache user
```

Comment:
Might be a nit, but I'd write this as `apache` to make it abundantly clear that this is actually the user named 'apache'.

Reply:
Make sense. I've added the `code` tags.